packages/perspective-viewer-highcharts/src/config/highcharts.plugin.esm.config.js

```diff
@@ -3,8 +3,9 @@ const common = require("@jpmorganchase/perspective/src/config/common.config.js")
 
 module.exports = Object.assign({}, common(), {
     entry: "./src/js/highcharts.js",
+    externals: [/^[a-z0-9\@].*$/],
     output: {
-        filename: "highcharts.plugin.js",
+        filename: "highcharts.plugin.esm.js",
         library: "perspective-view-highcharts",
         libraryTarget: "umd",
```

Comment:
Can this be changed to be `commonjs2` and the name to `cjs` since this is not `esm` target.